packages/react/src/components/Footer/FooterNav.js

```diff
@@ -45,13 +45,26 @@ function renderGroups(groups) {
   ));
 }
 
+/**
+ * @property propTypes
+ * @description Defined property types for component
+ * @type {{groups: shim}}
+ */
 FooterNav.propTypes = {
   groups: PropTypes.arrayOf(
     PropTypes.shape({
-      title: PropTypes.string.isRequired,
-      links: PropTypes.array.isRequired,
-    }).isRequired
-  ).isRequired,
+      title: PropTypes.string,
+      links: PropTypes.array,
+    })
+  ),
+};
+
+/**
+ * @property defaultProps
+ * @type {{groups: Array}}
+ */
+FooterNav.defaultProps = {
+  groups: null,
```

Comment:
Are these `null` defaults doing something? It's `undefined` by default. I think that would have the same outcome, unless I'm missing something?

Reply:
@photodow if a property is not required, linting requires to add in default prop values. We can either go with null, or an empty array. I think either achieves the same thing. But this to get rid of the console errors from tests since it was expecting these fields but was throwing an error when nothing was passed.

cc: @kennylam @annawen1

Reply:
Got it, is it necessary to remove `isRequired` then? They are required. If you don't include it the component won't render.

Reply:
Your tests were throwing console errors though when you don't. Perhaps adjusting the tests then to not throw the error but expect the error to happen?

Reply:
But per react it's supposed to throw those warnings, right? They were throwing those warnings because there are unit tests to test what happens when the props aren't included. Maybe we could use a method similar to #129 but more than just suppressing it we can see how many times it's being executed as well?

Reply:
That's what I was thinking, where we spy on console errors and would expect the error to throw if a user doesn't pass in the prop. Then we can set things back to required because the test is essentially doing its job.